server/service/integration_enterprise_test.go

```diff
@@ -8652,162 +8651,6 @@ func (s *integrationEnterpriseTestSuite) TestListHostSoftware() {
 	// TODO(mna): more advanced integration tests with Software Installers once the APIs are in place.
 }
 
-func (s *integrationEnterpriseTestSuite) TestHostSoftwareInstallResult() {
```

Comment:
This was moved to the mdm integration tests file, so that it could use the upload software installer.

Reply:
I think we put those in the MDM tests out of instinct but we'll eventually move all of them back here 😂